### ISSUE TITLE
Customer Service Hub- Accounts: NVDA is not announcing the menu control role and defined aria-label for the group of menuitems while navigating the menuitem.

### ISSUE THREAD

**Issue by msftedad** [nvaccess/nvda#13299]
![NVDA is not announcing menu information and aria-label with code](https://user-images.githubusercontent.com/93735775/152076440-32509b65-cf74-4ffb-9dee-d968b3cdd326.png)
![NVDA is not announcing menu information and aria-label](https://user-images.githubusercontent.com/93735775/152076444-1bc2013b-fe7b-4282-9e7a-4b2b080c8a34.png)

### Steps to reproduce:
1. Turn on NVDA.
2. Open URL: https://www.office.com/apps?auth=2 and Login with valid credentials.
3. Tab to 'Customer Service Hub' business app and activate it.
4. Tab to 'Accounts' control in left navigation.
5. Select any account.
6. Tab to 'Related' tab and activate it.
7. Select 'Cases' from the expanded list.
8. Now navigate to 'Refresh' control under the added 'Cases' tab.
9. Verify that NVDA does not announce the role menu and aria-label defined for the complete list of menuitems.

### Actual behavior:
NVDA is not announcing the menu control role and defined aria-label for the group of menuitems while navigating the menuitem.

### Additional Note:
1. Issue is not reproducible with JAWS and Narrator.
2. Issue is reproducible for other controls as well defined under menubar.

### Expected behavior:
NVDA should announce the menu control role and defined aria-label for the group of menuitems while navigating the menuitem.

### System configuration
Edge: Version 97.0.1072.76

#### NVDA installed/portable/running from source:
Installed

#### NVDA version:
Version 2021.3.1

#### Windows version:
OS: Windows 11 (Version 21H2 Build 22000.434)

#### Name and version of other software in use when reproducing the issue:
Application: Dynamics 365-> Customer Service Hub

#### Does the issue still occur after restarting your computer?
Yes

#### Have you tried any other versions of NVDA? If so, please report their behaviors.
No

**Comment by seanbudd** [comment]
- To be able to reproduce this issue, we need a code sample. Provide a minimal HTML sample to reproduce this using [codepen](https://codepen.io/).
- This is very likely a problem with the implementation of the application/website, as a starting point please submit feedback to the authors to let them know about the problem.

Kind Regards,
NV Access Software Developers
